packages/components/src/List/ListComposition/LazyLoadingList/LazyLoadingList.component.js

```diff
@@ -1,19 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { InfiniteLoader } from 'react-virtualized';
+import debounce from 'lodash/debounce';
 
 import VList from '../VList';
 import { useListContext } from '../context';
 
 const DEFAULT_THRESHOLD = 5;
 const DEFAULT_MIN_BATCH_SIZE = 20;
+const DEFAULT_DEBOUNCE_DELAY = 0;
 
 function LazyLoadingList(props) {
-	const { rowCount, threshold, minimumBatchSize, loadMoreRows, ...restProps } = props;
+	const { rowCount, threshold, minimumBatchSize, loadMoreRows, debounceDelay, ...rest } = props;
 	const { collection } = useListContext();
 
 	const isRowLoaded = ({ index }) => collection && collection[index];
 
+	let loadMoreRowsCallback = loadMoreRows;
```

Comment:
Should `loadMoreRowsCallback` not be passed to `<InfiniteLoader` `loadMoreRows` props?

Reply:
You're right, went to quick on that 🙃